amplify/backend/function/getUserReputation/src/index.js

```diff
@@ -0,0 +1,49 @@
+const { getColonyNetworkClient, Network } = require('@colony/colony-js');
+const {
+  providers,
+  utils: { Logger },
+} = require('ethers');
+
+Logger.setLogLevel(Logger.levels.ERROR);
+
+const ROOT_DOMAIN_ID = 1; // this used to be exported from @colony/colony-js but isn't anymore
+const RPC_URL = 'http://network-contracts.docker:8545'; // this needs to be extended to all supported networks
+
+/**
+ * @type {import('@types/aws-lambda').APIGatewayProxyHandler}
+ */
+exports.handler = async (event) => {
+  const input = event.arguments?.input;
+  const colonyAddress = input?.colonyAddress;
+  const walletAddress = input?.walletAddress;
+  const domainId = input?.domainId;
+  const rootHash = input?.rootHash;
+
+  const provider = new providers.JsonRpcProvider(RPC_URL);
+
+  const {
+    etherRouterAddress: networkAddress,
+    // eslint-disable-next-line global-require
```

Comment:
I'd also disable this rule in the function-level eslint, since we use it in all functions basically.